core/services/vrf/delegate.go

```diff
@@ -74,6 +75,11 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, jb job.Job) ([]job.Servi
 	if jb.VRFSpec == nil || jb.PipelineSpec == nil {
 		return nil, errors.Errorf("vrf.Delegate expects a VRFSpec and PipelineSpec to be present, got %+v", jb)
 	}
+	marshalledJob, err := json.MarshalIndent(jb.VRFSpec, "", " ")
+	if err != nil {
+		return nil, err
+	}
+	d.lggr.Debugw("Creating services for job spec", "job", string(marshalledJob))
```

Comment:
Can consider adding more tags like key hashes, max gas price, etc to make it easier to find within the logs.

Reply:
makes sense. so key hash can be derived from "uncompressedKey" in the job spec. max gas price is already being printed elsewhere: `Key = '0xd13b46290f61dB3E6638bE2746e88F0c04AB883F'\n\n[EVM.KeySpecific.GasEstimator]\nPriceMax = '200 gwei'`

Reply:
Right. Anything printed can be found as a regex matching the log since tags are not indexed anyway.